### DISCUSSION TITLE
Examples using global state

### DISCUSSION THREAD

**Discussion by scottroot** [avkonst/react-use-state-x#404] [Q&A]
Hello, I have become a big user of hookstate as it's expanded my ability to tackle complex use cases.  I keep wanting to use the global hookstate() functionality though have not yet figured out the best way to do so. 

I was wondering if there are any other examples or descriptions of use cases for this?

**Comment by avkonst** [comment]
It works and used the same way as local state. Just declared in a variable outside of a react component. https://hookstate.js.org/docs/global-state. The question about local vs global state is equivalent to long lived debate "local variable vs global variable". It is simple as that.
